articles/email/custom.md

```diff
@@ -1,22 +1,24 @@
-# Take control of the complete Email flow
+# Custom Email Handling
 
-The default email flow in Auth0 can address the requirements of most applications, but there might still be cases where you might need more flexibility:
+The default email flow in Auth0 can address the requirements of most applications, but there may be instances where more flexibility is required. For example:
 
- - Localization
- - Calculated **Redirect To** url based on user/tenant/...
- - Different email templates per application/tenant/...
+ * Localization
+ * Custom **Redirect To** URLs based on the user or tenant
+ * Different email templates per application or tenant
 
-Our extensiblity points, together with our API allow you to completely take over the email flow and control when and how emails are being sent out.
+The Auth0 API exposes endpoints that will allow you to completely manage email flow and control when and how emails are sent.
 
-To get started you will need to disable the Emails [in the dashboard](${uiURL}/#/emails) after which you will be able to implement the different email flows in your own environment.
+To begin, you will need to disable automatic emails by deselecting **Status** under the **Verification Email** and **Welcome Email** tabs on the [Email Templates](${uiURL}/#/emails) page of the Auth0 dashboard.
```

Comment:
yes, otherwise both auth0 and the customer implementation will send them. You only want the custom email to be sent.